src/app/board/board.component.html

```diff
@@ -1,8 +1,8 @@
 <app-header></app-header>
-<div class="container-fluid container-cards-pf">
+<div id="board_topWorkItems" class="container-fluid container-cards-pf">
   <h3>Top WorkItems</h3>
   <div class="grid grid-pad">
-    <div *ngFor="let workItem of workItems" (click)="gotoDetail(workItem)" class="col-1-4">
+    <div *ngFor="let workItem of workItems" (click)="gotoDetail(workItem)" class="col-1-4" id="board_topWorkItemList">
```

Comment:
Thinking 'semantic html', we need to avoid position reference in the ID too. Based on the final wireframe, the board might appear anywhere. Say, the board appears at the bottom, in that case board_topWorkItemList would not make sense.